### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jenschelkopf**
Thanks for this awesome work 🎉

Here's a few suggestions for updating the README. The goals of this PR are:

* Reduce the emphasize on 'centrally managed.' In my conversations with GitHub customers, many expect 'centrally managed' to be templates. They expect developers to extend and customize these templates, and to be able to push down updates. To help reduce confusion between this and the future Centrally Managed Workflow Templates feature, I've made a few edits and reduced the emphasize on management.
* Add language to help a team determine whether this app is right for them, especially those with security concerns

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Organization workflows app
 
-This GitHub app allows you to centrally manage and run multiple [GitHub Actions](https://github.com/features/actions) workflows across multiple repositories. Currently this is a limitation in GitHub Actions, as it only allows you to configure and manage Actions workflows on a repository level. This app helps you - for example - to centrally define central workflows for linting, compliance checks, and more.
+This GitHub app allows you to run [GitHub Actions](https://github.com/features/actions) workflows across multiple repositories, which is [not yet natively suppoprted](https://github.com/github/roadmap/issues/52). This app helps you - for example - to create a single workflow definition that is used for linting, compliance checks, and more.
+
+This app may not be right for you if:
+* You require production quality. This app is considered expeirmental and runs on a free Heroku dyno
```
Comment (rd-matthias-jambor) [inline README.md]: There is a typo at "expeirmental"

Comment (SvanBoxel) [reply README.md]: This line can be removed altogether. All resources are hosted on scalable Azure instances now. 👍🏽

Comment (jenschelkopf) [reply README.md]: Removed!

**Comment by jenschelkopf** [comment]
@SvanBoxel Could you take another look at this, please?

**Comment by SvanBoxel** [comment]
Love it, thank you @jenschelkopf 🙇

**Pull request merged**